packages/components/pie-text-input/CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
-# @justeattakeaway/pie-input
+# @justeattakeaway/pie-text-input
```

Comment:
I think in this file it might be worth keeping the previous entries with their prior naming, and then adding a manual entry to say that it was renamed from v0.19.0 onwards (assuming this won't break things going forwards).